pkg/pgmodel/end_to_end_tests/promql_label_endpoint_test.go

```diff
@@ -75,28 +77,39 @@ func TestPromQLLabelEndpoint(t *testing.T) {
 		r := pgmodel.NewPgxReader(readOnly, nil, 100)
 		queryable := query.NewQueryable(r.GetQuerier())
 
-		apiConfig := &api.Config{}
+		apiConfig := &api.Config{
+			AllowedOrigin: regexp.MustCompile(".*"),
+		}
 		labelNamesHandler := api.Labels(apiConfig, queryable)
 		labelValuesHandler := api.LabelValues(apiConfig, queryable)
 
 		router := route.New()
 		router.Get("/api/v1/label/:name/values", labelValuesHandler.ServeHTTP)
 		router.Get("/api/v1/labels", labelNamesHandler.ServeHTTP)
 
-		apiURL := fmt.Sprintf("http://%s:%d/api/v1", testhelpers.PromHost, testhelpers.PromPort.Int())
+		ts := httptest.NewServer(router)
```

Comment:
Is this the actual change that fixed the issue? If so I would explain in the commit message a bit more....

Reply:
Yep, will update it.